selfdrive/car/gm/carcontroller.py

```diff
@@ -25,13 +34,15 @@ def __init__(self, dbc_name, CP, VM):
     self.lka_icon_status_last = (False, False)
 
     self.params = CarControllerParams()
+    self.pitch = FirstOrderFilter(0., 0.09 * 4, DT_CTRL * 4) # runs at 25 Hz
```

Comment:
if we only use the filter, ensure that going from min to max pitch doesn't violate your selected rate limit above